workers/unity/Assets/Playground/Editor/SnapshotGenerator/SnapshotGenerator.cs

```diff
@@ -77,7 +79,7 @@ private static void AddCubeGrid(Snapshot snapshot, int cubeCount)
                     }
 
                     // Exit when we've hit our cube limit
-                    if (--cubesToSpawn <= 0)
+                    if (cubesToSpawn-- <= 0)
```

Comment:
Was this broken?

Reply:
Yes, we were previously spawning 15 cubes instead of 16.